### ISSUE TITLE
dask-jobqueue for Fujitsu HPC

### ISSUE THREAD

**Issue by MichaelChaoLi-cpu** [dask/dask-jobqueue#573]
In the dask-jobqueue, there are a lot of APIs for supercomputer structures. 
However, there is no Fujitsu HPC. Fujitsu structure is widely used in Japan. In the job script, it should be #PJM. Is it possible?

**Comment by guillaumeeb** [comment]
Hi @MichaelChaoLi-cpu,

Thanks for rising an issue here! 

It's always possible to add new job queuing system implementation in dask-jobqueue. However it needs someone that knows well the underlying job scheduler and can devote time to this implementation. Would you want to give it a try?

**Comment by MichaelChaoLi-cpu** [comment]
Hi @guillaumeeb 

Thanks for replying. I am not very familiar with this, but I will try to ask the supercomputer sysadmin to support this.

Let me contact him.

Anyway, thanks for your time!

**Comment by MichaelChaoLi-cpu** [comment]
Hi @guillaumeeb 

In fact, I have a problem to solve. Can you provide some information about this?
I am training a sklearn.ensemble.RandomForestRegressor
```
from sklearn.datasets import make_regression
X, y = make_regression(n_samples = 6300000, n_features = 55, random_state=1) ### size is 6,300,000, huge!

from sklearn.ensemble import RandomForestRegressor
model = RandomForestRegressor(n_estimators=1000, oob_score=True, random_state=1, n_jobs=36)
model.fit(X, y)
```
I tried it on one node (180GB memory, 36 cores) of a cluster (I have 16 these nodes in this cluster), but it failed. However, if I use the node with 360GB and 36 cores, it can run smoothly (This is a different type and I just have one).
So. memory is the limitation. Furthermore, training the model on one node with 36 cores is slow, even though the training works on the 360GB node. Of course, I hope that I can use all the cores. 

Could Dask distribute the data into four nodes (180GB * 4) and run the program in parallel? Using all 180GB*4 and 36 cores*4.

Thank you. (This problem has confused me for a long time. I would really appreciate it if you or someone you know could give me some hints)

Best regards

**Comment by guillaumeeb** [comment]
@MichaelChaoLi-cpu, I'm going to delete your last message here. I see you posted the question on Dask Discourse which is really the good place for that. Moreover, this question didn't have anything to do with your first question.
